spec/debug_logging/class_notifier_spec.rb

```diff
@@ -20,10 +20,10 @@
         complete_notified_klass.k_with_dsplat(a: 'a')
       end
       expect(output).to match(/k_with_dsplat.log/)
-      expect(output).to match(/payload={:args=>\[{:a=>"a"}\]}/)
+      expect(output).to match(/payload={:debug_args=>\[{:a=>"a"}\]}/)
       expect(complete_notified_klass.instance_variable_get(DebugLogging::Configuration.config_pointer('k', :k_with_dsplat))).to receive(:log)
       expect(@events).to contain_exactly(
-        have_attributes(name: 'k_with_dsplat.log', payload: { args: [{ a: 'a' }] })
+        have_attributes(name: /k_with_dsplat.log/, payload: { debug_args: [{ a: 'a' }] })
```

Comment:
Prefer to use the string over the regex, because we want exact matches here.  If we can match on a more precise expectation we should.  Same goes for other string => regex updates elsewhere in specs.

Reply:
I switched out all the matches on the regex with exact string matches throughout all the notification-related specs:

`spec/debug_logging/class_notifier_spec.rb`
`spec/debug_logging/instance_notifier_spec.rb`
`spec/debug_logging/log_subscriber_spec.rb`